docs/Agent/installation/rpm_based_installation.md

```diff
@@ -1 +1,40 @@
 # RPM based installation
+
+The pbench agent requires the installation of some generic bits, but it also requires some localization: it needs to know where to send the results for storage and analysis and it needs to be able to authenticate to the results server.
+
+The generic bits are packaged as an RPM, available from [COPR](https://copr.fedorainfracloud.org/coprs/ndokos). We try to build a new release every month or so. Pbench-agent is build for  all major releases of Fedora, RHEL, CentOS and openSUSE.
+
+The localization bits are of course, specific to a particular installation. Internally, we make them available through an ansible playbook. We used to make them available through an internal RPM, but we have deprecated that method: we no longer build internal RPMs. Other installations may use different methods to make them available. Consult your local pbench guru for help.
+
+In the following, we describe how to install pbench-agent using an RPM.
+
+## Setup
+
+1. Pbench-agent rpm is packaged in [COPR](https://copr.fedorainfracloud.org/coprs/ndokos). Choose a release(prefeably the latest) and enable the repo.
+
+	```console
+	dnf copr enable ndokos/pbench-0.72
+	```
+
+    :::{note}
+	- At the time of writing this doc  [pbench-agent-0.72](https://copr.fedorainfracloud.org/coprs/ndokos/pbench-0.72) is the lastest release.
```

Comment:
since I mentioned the hardcoded version in the doc for repo enablement, I felt that it must be mentioned.
else we have to always update the page with the latest version.
or 
if we don't want to specify any version details let's put placeholders in place and get rid of the note section
like 
```console
dnf copr enable ndokos/pbench-<version>
```

Reply:
fixed